index.html

```diff
@@ -1,7 +1,34 @@
 <!DOCTYPE html>
 <html>
-    <head></head>
-    <body>
-        Hello, World
-    </body>
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Simple Landing Page</title>
+    <meta name="viewport" content="width=device-width, initian-scale=1">
+    <link rel="stylesheet" hret="style.css">
+</head>
+<body>
+    
+    <!---------- Header section ------------>
+
+    <div class="container">
+        <header>
+           <nav>
+            <img src="img/logo.png"   class="logo">
+                <ul>
+                    <li><a href="#home">Home</a></li>
+                    <li><a href="#how">How it work</a></li>
+                    <li><a href="#organic food">organic food</a></li>
+                    <li><a href="#about">About farm</a></li>
+                    <li><a href="#animals">Animals</a></li>
+                    <li><a href="@products">Products</a></li>
+                    <li><a href="contacts">ontacts</a></li>
```

Comment:
Pay attention to the last two rows, also typos

Reply:
I found it, everything worked out:)